src/System.Runtime.Extensions/tests/System/IO/Path.Combine.cs

```diff
@@ -157,9 +162,6 @@ public static void ContainsInvalidCharWithRootedAfterArgumentNull()
         public static void ContainsInvalidCharWithRootedAfterArgumentNull_Windows()
         {
```

Comment:
Can these removed tests simply be moved to somewhere that'll validate they pass rather than fail?